core/src/test/scala/zio/config/ListsCornerCasesTest.scala

```diff
@@ -7,279 +7,277 @@ import zio.config.ReadError.{ FormatError, ListErrors, ZipErrors }
+          )
+        )
+      },
+      test("collect errors from list elements") {
+        case class Cfg(a: String, b: List[String])
+
+        val cCfg =
+          (string("a") |@| nested("b")(list(string))).to[Cfg]
+
+        val res = read(
+          cCfg from ConfigSource.fromPropertyTree(
+            Record(
+              Map(
+                "a" -> Leaf("sa"),
+                "b" -> Sequence(
+                  Record[String, String](Map.empty) :: Sequence(Nil) :: Nil
                 )
-              ),
-              "tree",
-              LeafForSequence.Valid
-            )
+              )
+            ),
+            "tree",
+            LeafForSequence.Valid
           )
+        )
 
-          assert(res)(isLeft(hasField("size", _.size, equalTo(2))))
-        },
-        test("accumulates all errors") {
-          case class Cfg(a: List[Boolean], b: List[Int])
+        assert(res)(isLeft(hasField[ReadError[String], Int]("size", _.size, equalTo(2))))
+      },
+      test("accumulates all errors") {
+        case class Cfg(a: List[Boolean], b: List[Int])
 
-          val cCfg = (nested("a")(list(boolean)) |@| nested("b")(list(int)))(Cfg, Cfg.unapply)
+        val cCfg = (nested("a")(list(boolean)) |@| nested("b")(list(int))).to[Cfg]
 
-          val res = read(
-            cCfg from ConfigSource.fromPropertyTree(
-              Record(
-                Map(
-                  "a" -> Sequence(Leaf("true") :: Leaf("lorem ipsum") :: Nil),
-                  "b" -> Sequence(Leaf("one") :: Leaf("2") :: Nil)
+        val res = read(
+          cCfg from ConfigSource.fromPropertyTree(
+            Record(
+              Map(
+                "a" -> Sequence(Leaf("true") :: Leaf("lorem ipsum") :: Nil),
+                "b" -> Sequence(Leaf("one") :: Leaf("2") :: Nil)
+              )
+            ),
+            "tree",
+            LeafForSequence.Valid
+          )
+        )
+
+        val expected =
+          ZipErrors(
+            List(
+              ListErrors(
+                List(
+                  FormatError(List(Key("a"), Index(1)), "Provided value is lorem ipsum, expecting the type boolean")
                 )
               ),
-              "tree",
-              LeafForSequence.Valid
+              ListErrors(List(FormatError(List(Key("b"), Index(0)), "Provided value is one, expecting the type int")))
             )
           )
 
-          val expected =
-            ZipErrors(
-              List(
-                ListErrors(
-                  List(
-                    FormatError(List(Key("a"), Index(1)), "Provided value is lorem ipsum, expecting the type boolean")
-                  )
-                ),
-                ListErrors(List(FormatError(List(Key("b"), Index(0)), "Provided value is one, expecting the type int")))
-              )
-            )
-
-          assert(res)(isLeft(equalTo(expected)))
-        }
-      )
+        assert(res)(isLeft(equalTo(expected)))
+      }
     )
+}
```

Comment:
Difficulty knowing what exactly changed.. But I hope its all good.

Reply:
Actually I think I mostly only changed the apply/unapply pairs to `to` but I guess that made scalafmt reformat things a bit as the lines became shorter..

Reply:
Actually if you "hide whitespace changes" on the github ui, then it shows it nicely. Not sure why, I guess it was scalafmt'd before too..